src/helpers/firebaseHelpers.js

```diff
@@ -1,8 +1,8 @@
 export const parseData = (querySnapshot) =>
   querySnapshot.docs.reduce((acc, doc) => {
-    const { productName, userToken, timeFrame, lastPurchaseDate } = doc.data();
+    const { productName, listToken, timeFrame, lastPurchaseDate } = doc.data();
```

Comment:
I think it could be much better if you might create an independent function called `getDataFromDoc(doc)`

```js
export const getDataFromDoc = (doc) => {
     const {id} = doc;
     const { productName, listToken, timeFrame, lastPurchaseDate } = doc.data();
     return {id, productName, listToken, timeFrame, lastPurchaseDate}
}
```

```suggestion
    const docValues = getDataFromDoc(doc);
```

What do you think about it ^^?